Changelog.md

```diff
@@ -14,6 +14,11 @@ Enhancements:
   `RSpec::Matchers#respond_to?` and `RSpec::Matchers#method` handle
   dynamic predicate matchers. (Andrei Botalov, #751)
 
+Bug Fixes:
+
+* Exclude Ranges from being enumerable when generating matcher
```

Comment:
Ranges are still `Enumerable` but we are choosing not to enumerate the.  Maybe "Exclude Ranges from being enumerated when generating matcher descriptions and failure messages" is better?